jupyter_book_to_htmlbook/helpers.py

```diff
@@ -0,0 +1,6 @@
+def base_soup(element):
```

Comment:
Praise: I love the idea of refactoring pieces of the code to a helpers module as the package grows.

Reply:
TY!